packages/lab/scripts/build.mjs

```diff
@@ -0,0 +1,57 @@
+import esbuild from "esbuild";
```

Comment:
any reason we couldn't reuse these scripts? Seem to be identical between core and lab, with a few arguments

Reply:
yeah probably...where would you keep them?

Reply:
Something like `/tooling/css-bundle/build.mjs`?